src/serializer/modules.js

```diff
@@ -461,6 +393,9 @@ export class Modules {
       if (escapes) return undefined;
 
       return compl;
+    } catch (err) {
+      if (err instanceof FatalError) return undefined;
```

Comment:
It got uncovered by throwing FatalError for all failures and therefore needs fixing here.